netsuite/src/TypeScript/models/match-group/match-group-constants.ts

```diff
@@ -0,0 +1,24 @@
+export enum FIELD {
+    internalId = 'internalid',
+    matchingDate = 'custrecord_eqtmt_match_gr_date',
+    matchingStatus = 'custrecord_eqtmt_match_gr_status',
+    matchingCreator = 'custrecord_eqtmt_match_gr_creator',
+    creditDebitDiff = 'custrecord_eqtmt_match_gr_deb_cre_diff',
+}
+
+export enum TYPE {
+    matchingGroup = 'customrecord_eqtmt_matched_group'
+}
```

Comment:
```suggestion
export enum Type {
    MATCHED_GROUP = 'customrecord_eqtmt_matched_group'
}
```

Vi hade tidigare en enum RecordType för alla netsuite idn för både våra customrecords och netsuites egna records. Skulle det vara skönt att använda igen? Kanske inte behovet finns ännu. tänker högt.